backend/tests/conftest.py

```
@@ -0,0 +1,50 @@
+from fastapi.testclient import TestClient
+import sys
+from pathlib import Path
+from fastapi import Depends
+sys.path.append(str(Path(__file__).resolve().parents[1]))
+from database.session import engine, Base, SessionLocal
+from main import app
+
+# Import all models to register with Base
+from models.user import User
+from models.connection import Connection
+from models.chat import Chat  # Verify this matches your chat.py
```

Ruff (0.8.2)

`models.chat.Chat` imported but unused

Remove unused import: `models.chat.Chat`

(F401)